### PR TITLE
bump to 2.7.0

### PR DESCRIPTION
**Pull request by ruanyl**
Signed-off-by: Yulong Ruan <ruanyl@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-ml-commons-dashboards.release-notes-2.7.0.0.md
Comment (wanglam) [inline release-notes/opensearch-ml-commons-dashboards.release-notes-2.7.0.0.md:1]: Could we add a  Bug Fixes section include [this PR](https://github.com/opensearch-project/ml-commons-dashboards/pull/146)?

File: release-notes/opensearch-ml-commons-dashboards.release-notes-2.7.0.0.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Version 2.7.0.0 Release Notes
+
+Compatible with OpenSearch 2.7.0
+
+### Breaking Changes
+
+* ml-commons introduced a API breaking change in 2.7 which changed the API response data, FE is updated accordingly. With this change, FE will no longer compatible with OpenSearch version < 2.7 ([#154](https://github.com/opensearch-project/ml-commons-dashboards/pull/154))
```
Comment (wanglam) [inline release-notes/opensearch-ml-commons-dashboards.release-notes-2.7.0.0.md:7]: Could we adjust the description about this breaking change according [the conventional](https://github.com/opensearch-project/opensearch-plugins/blob/main/RELEASE_NOTES.md)?

Comment (ruanyl) [reply release-notes/opensearch-ml-commons-dashboards.release-notes-2.7.0.0.md:7]: Could you elaborate a bit?

File: release-notes/opensearch-ml-commons-dashboards.release-notes-2.7.0.0.md
Comment (ruanyl) [reply release-notes/opensearch-ml-commons-dashboards.release-notes-2.7.0.0.md:1]: Yep, we should. Feel free to create a PR for that 👍

Comment (wanglam) [reply release-notes/opensearch-ml-commons-dashboards.release-notes-2.7.0.0.md:1]: Do we need to add this PR to the Maintenance section? Just find some [bump version example](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/main/release-notes/opensearch-anomaly-detection-dashboards.release-notes-2.4.0.0.md) in ad plugin.

File: release-notes/opensearch-ml-commons-dashboards.release-notes-2.7.0.0.md (hunk 1 above)
Comment (wanglam) [reply release-notes/opensearch-ml-commons-dashboards.release-notes-2.7.0.0.md:7]: I can't find the breaking changes example in other plugins. The description should short and use capitalized for first letter. I may change to 
```
* Change model_state according ml-commons 2.7 ([#154](https://github.com/opensearch-project/ml-commons-dashboards/pull/154))
```

Comment (ruanyl) [reply release-notes/opensearch-ml-commons-dashboards.release-notes-2.7.0.0.md:7]: > The description should short

I think the description doesn't have to be "short", the description can be compact but at least it should shortly explain what's the change and why making this change, especially for breaking changes.

Also, I personally don't feel the breaking change example in documentation is a good example :D

> use capitalized for first letter

It should use capitalized letter when it applicable, but in this case, the description starts with `ml-commons` which is a component name

**Review by wanglam — APPROVED** [review]

**Comment by wanglam** [comment]
@ruanyl Do we need to backport to 2.x?

**Comment by ruanyl** [comment]
> @ruanyl Do we need to backport to 2.x?

This is targeting 2.x, do you mean `main` branch?

**Comment by wanglam** [comment]
> > @ruanyl Do we need to backport to 2.x?
> 
> This is targeting 2.x, do you mean `main` branch?

Yes. I think the release-notes should be sync to `main` branch.

**Pull request merged**